src/Tribe/Views/V2/Template/Title.php

```diff
@@ -132,7 +132,7 @@ public function build_title( $current_title = '', $depth = true, $sep = null ) {
 		} elseif ( $context->is( 'single' ) && $context->is( 'event_post_type' ) ) {
 			// For single events, the event title itself is required
 			$title = get_the_title( $context->get( 'post_id' ) );
-		} else if ( $event_date && count( $posts ) ) {
+		} else if ( count( $posts ) ) {
```

Comment:
How does this affect first pages?
Do we still see "Upcoming Events" ever - if we have events?